File: 알고리즘/전승민/phoneketmon/phoneketmon.java
import java.util.HashSet;
import java.util.Set;

public class phoneketmon {
    public int solution(int[] nums) {
        int answer = 0;
        Set<Integer> set = new HashSet();
        for(int i : nums) {
            set.add(i);
        }
        if(nums.length/2>set.size()) {
            answer = set.size();
        } else {
            answer = nums.length/2;
        }
        return answer;
    }
}

File: 알고리즘/권하경/20-07-09/phoneketmon.js
function solution(nums) {
  let arr = [];
  for(let i=0;i<nums.length;i++){
    if(!arr.includes(nums[i])) arr.push(nums[i]);
  }
  return arr.length<=nums.length/2 ? arr.length : nums.length/2
}
